example/spec/parallel/ingestGranule/IngestGranuleSuccessSpec.js

```diff
@@ -1087,14 +1089,14 @@ describe('The S3 Ingest Granules workflow', () => {
         // Check that the granule was removed
         await waitForConceptExistsOutcome(cmrLink, false);
         const doesExist = await conceptExists(cmrLink);
-        expect(doesExist).toEqual(false);
+        expect(doesExist).toEqual(true); // because of dummied conceptExists
       });
 
       it('applyWorkflow PublishGranule publishes the granule to CMR', async () => {
         failOnSetupError([beforeAllError, subTestSetupError]);
 
         const existsInCMR = await conceptExists(cmrLink);
-        expect(existsInCMR).toEqual(false);
+        expect(existsInCMR).toEqual(true); // because of dummied conceptExists
```

Comment:
That's a fun consequence of stubbing this out.  The simplest edit seems fine given context, but I think we need to self-refer to this PR in the comment with this sort of thing.

```suggestion
        expect(existsInCMR).toEqual(true); // because of stubbed conceptExists  due to https://github.com/nasa/cumulus/pull/3544/  / removing CMR query from integration tests due to <ticket> 
```